main.js

```diff
@@ -0,0 +1,85 @@
+const inViewport = require('in-viewport');
+
+//find all convo cards
+const convoCards = document.getElementsByClassName('conversation');
+
+//for each card, define inViewport callback function
+[].slice.call(convoCards).forEach(card => inViewport(card, { offset: -200 }, function() {
```

Comment:
Instead of using anonymous functions in your `forEach` loops, it would look a little cleaner to separate them out.